content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -598,6 +597,18 @@ The number of datagrams dropped by the DogStatsD client.
 : **Metric type**: count<br>
 The number of datagrams dropped because the DogStatsD client queue was full.
 
+`datadog.dogstatsd.client.aggregated_context`
+: **Metric type**: count<br>
+The number of contexts aggregated when client side aggregation is enabled. Starting with version `v2.11.0`.
+
+`datadog.dogstatsd.client.aggregated_context_by_type`
+: **Metric type**: count<br>
+The number of contexts aggregated by type when client side aggregation is enabled. Starting with version `v2.13.0`. The metric is enabled by default starting `v3.0.0` but requires `enableDevMode(true)` for `v2.13.0`. The metric is tagged with either `metrics_type:gauge`, `metrics_type:count`, `metrics_type:set`, `metrics_type:histogram` nor `metrics_type:distribution`.
+
+`datadog.dogstatsd.client.metrics_by_type`
+: **Metric type**: count<br>
+The number of metrics by type sent to the DogStatsD client by your application (before sampling). Starting with version `v2.13.0`. The metric is enabled by default starting `v3.0.0` but requires `enableDevMode(true)` for `v2.13.0`. The metric is tagged with either `metrics_type:gauge`, `metrics_type:count`, `metrics_type:set`, `metrics_type:histogram` nor `metrics_type:distribution`.
```

Comment:
What about:

```suggestion
The number of metrics sent to the DogStatsD client by your application tagged by type (before sampling). Starting with version `v2.13.0` when `enableDevMode(true)` is used and by default starting `v3.0.0`. The metric is tagged with either `metrics_type:gauge`, `metrics_type:count`, `metrics_type:set`, `metrics_type:histogram` nor `metrics_type:distribution`.
```

Also do the C# client support timing metrics ? Since the interface will list all tags I don't know if it's necessary to list all metric types. WDYT ?

Reply:
This section is about the java client.

Reply:
Well, same question for java then

Reply:
cc @vickenty

Reply:
Java client does not have timing metrics. I think we can mention only the tag key (`metrics_type`) without listing every possible metric type here.